tools/dependencies/known-dependencies.txt

```diff
@@ -606,11 +608,12 @@ protobuf-java-2.5.0.jar
 protobuf-java-3.17.1.jar
 rank-eval-client-6.3.1.jar
 rank-eval-client-7.5.1.jar
+rank-eval-client-7.15.2.jar
 re2j-1.1.jar
-reactive-streams-1.0.2.jar
 reactive-streams-1.0.3.jar
 reindex-client-6.3.1.jar
 reindex-client-7.5.1.jar
+reindex-client-7.15.2.jar
```

Comment:
Many versions of sweet jars have been added, can you unify them?

Reply:
> Many versions of sweet jars have been added, can you unify them?

The newly added jar is detected by the checkLicense script. 
I found that there are many cases of multiple versions of one jar. I suggest to optimize this problem in the next PR. The current PR is to add web mvc unit tests.

Reply:
if you just add  test, then it is best not to modify the dependencies related.

Reply:
The `Check Dependencies Licenses` step will report an error.

Reply:
> if you just add test, then it is best not to modify the dependencies related.

ok, I will fix the dependencies in this PR.

Reply:
> Many versions of sweet jars have been added, can you unify them?

There are multiple versions of a jar partly because there are multiple versions of the same connector, such as connector-elasticsearch6 and connector-elasticsearch7.